package.json

```diff
@@ -19,7 +20,8 @@
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
     "node-fetch": "^2.6.1",
-    "pg": "^8.3.3"
+    "pg": "^8.3.3",
```

Comment:
are we still using `pg` somewhere? if not, we can probably safely delete it

Reply:
No, we are not.
Fixed in the latest commit.